scripts/run-spark.py

```diff
@@ -101,6 +115,8 @@ def get_spark_args(args):
         ret.append("-c spark.shuffle.service.enabled=true")
     if args.push_based_shuffle:
         ret.append("-c spark.shuffle.push.enabled=true")
+    # Spark default parallelism
+    ret.append(f"-c spark.default.parallelism={args.num_mappers}")
```

Comment:
Does this have any effect on the sort parallelism? If not then let's not add this here.